verifier/check_known_issues.py

```diff
@@ -117,6 +121,32 @@ def unsupported_unit_quarter(test):
     return None
 
 
+def dt_check_for_alternate_long_form(actual, expected):
+    # For datetime_fmt, check if the word for "at" is inserted
+    # in the actual vs. the expected.
+    at_inserts = ['at ', 'এ ', 'lúc ', ',', ' এ']
+    replacements = {',': ' at', '،': ' في',
+                    }
+
+    sm = SequenceMatcher(None, expected, actual)
```

Comment:
This is a much more complicated known-issue catcher than I was expecting. I was expecting that we would put a blanket "known issue" on all failing inputs where the dateTimeFormatType is "standard".

Reply:
OK, I'll take a look at the failures to see if this will capture the substitution of "," with " at" and similar replacements in other languages.